.github/metallb-config.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  name: example
+  namespace: metallb-system
+spec:
+  addresses:
+    - 172.19.200.200-172.19.200.210
+---
+apiVersion: metallb.io/v1beta1
+kind: L2Advertisement
+metadata:
+  name: empty
+  namespace: metallb-system
```

Comment:
Where those resources are created? I don't see `kubectl apply -f` or any other place where this file is referenced.

Reply:
Not yet, this is still in testing locally phase, will pick this PR back up in a few.